main.go

```diff
@@ -3,44 +3,122 @@ package main
 import (
 	"errors"
 	"fmt"
+	"log/slog"
 	"net/http"
 	"os"
+	"strings"
+	"text/template"
 
 	"github.com/Fesaa/ical-merger/config"
 	"github.com/Fesaa/ical-merger/ical"
 	"github.com/Fesaa/ical-merger/log"
 	"github.com/Fesaa/ical-merger/server"
 )
 
+const motd = `
+=======================================
+Listen on: {{.Host}}
+Broadcasting notifications to: {{.Config.WebHook}}
+Publishing:
+{{- range .Config.Sources }}
+  {{.Name}}: {{$.Host}}/{{.EndPoint}}.ics
+{{- end }}
+=======================================
+`
+const icsPrefix = ".ics"
```

Comment:
This seems to be only used in tests? Can you move to the test file, thanks.

Reply:
Done